bakery/src/tests/test_cli.js

```diff
@@ -174,6 +174,17 @@ test('stable flow in pdf and distribution pipeline', async t => {
   const assembleMetaResult = await completion(assembleMeta)
   t.truthy(fs.existsSync(`${outputDir}/${bookId}/assembled-book-metadata/${bookId}/collection.assembled-metadata.json`), formatSubprocessOutput(assembleMetaResult))
 
+  const link = spawn('node', [
```

Comment:
General question as far as adding this test case: Do you know whether the test book results in queries to archive? (e.g. are there external book links in `col30149`) If so, we may want to think about whether to include as part of these.